credentials/generate-revocation-set.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Generates a basic RevocationSet from TestNet
+# Usage:
+#     python ./credentials/generate-revocation-set.py --help
+
+import copy
+import os
+import re
+import subprocess
+import sys
+import json
+import base64
+
+import click
+import requests
+from click_option_group import RequiredMutuallyExclusiveOptionGroup, optgroup
+
+PRODUCTION_NODE_URL = "https://on.dcl.csa-iot.org:26657"
+PRODUCTION_NODE_URL_REST = "https://on.dcl.csa-iot.org"
+TEST_NODE_URL_REST = "https://on.test-net.dcl.csa-iot.org"
+
+
+def use_dcld(dcld, production, cmdlist):
+    return [dcld] + cmdlist + (['--node', PRODUCTION_NODE_URL] if production else [])
+
+
+def is_base64(sb):
+        try:
+                if isinstance(sb, str):
+                        # If there's any unicode here, an exception will be thrown and the function will return false
+                        sb_bytes = bytes(sb, 'ascii')
+                elif isinstance(sb, bytes):
+                        sb_bytes = sb
+                else:
+                        raise ValueError("Argument must be string or bytes")
+                return base64.b64encode(base64.b64decode(sb_bytes)) == sb_bytes
+        except Exception:
```

Comment:
I think ValueError would cover all of it

Reply:
Addressed in the latest commit.